controller/api/models.py

```diff
@@ -216,6 +218,17 @@ def _get_command(self, container_type):
             return '' if container_type == 'cmd' else 'start {}'.format(container_type)
 
     def log(self, message):
+        if self._get_logging_mode() == 'standard':
```

Comment:
It's probably not worth changing how the controller writes application event logs (scaling events, for instance) since that is / will be addressed in #3963 and #4347 

The only real change to the controller may be the ability to _retrieve_ logs from the logger via HTTP-- no need to write them via HTTP.